project_liberation/project_liberation/settings/__init__.py

```diff
@@ -0,0 +1,14 @@
+try:
```

Comment:
Is this something you have added or now Django auto generates this?

Reply:
This looks like my usual code for importing `local_settings.py`. It's not generated by Django.

@kbeker It would be more helpful if you had the files generated by Django and the custom changes made on top of them in separate commits. Then every change could have a description explaining why you're doing it. As it is now, it's very hard to see what is standard stuff and what is not. For example I just skimmed over the settings file and I have no idea if you added something there or not - checking that manually would be just too time consuming. Good split into commits is essential to make this review easy.

Reply:
Sure, I will change it. My bad :)